docs/machine-learning/resources/tasks.md

```diff
@@ -53,9 +104,53 @@ An [unsupervised machine learning](glossary.md#unsupervised-machine-learning) ta
 * Identifying customer segments and demographics to help build targeted advertising campaigns.
 * Categorizing inventory based on manufacturing metrics.
 
-## Anomaly detection (*coming soon*)
+### Clustering Learners
+
+The following learners are available for clustering tasks:
+
+* [KMeansPlusPlusTrainer](xref:Microsoft.ML.Trainers.KMeans.KMeansPlusPlusTrainer)
+
+## Anomaly detection
+
+This task creates an anomaly detection model by using Principal Component Analysis (PCA). PCA-Based Anomaly Detection helps you build a model in scenarios where it is easy to obtain training data from one class, such as valid transactions, but difficult to obtain sufficient samples of the targeted anomalies.
+
+Principal Component Analysis, which is frequently abbreviated to PCA, is an established technique in machine learning. PCA is frequently used in exploratory data analysis because it reveals the inner structure of the data and explains the variance in the data. PCA works by analyzing data that contains multiple variables. It looks for correlations among the variables and determines the combination of values that best captures differences in outcomes. These combined feature values are used to create a more compact feature space called the principal components.
```

Comment:
nit: abbreviated to --> abbreviated as

Reply:
I'd just do Principal Component Analysis (PCA) as indicated in the style guide:
https://docs.microsoft.com/en-us/style-guide/acronyms#if-you-have-to-use-an-acronym-also-spell-out-the-term-for-clarity

In general, include the acronym in parentheses following the spelled-out term.